code/modules/mob/living/simple_animal/hostile/venus_human_trap.dm

```diff
@@ -189,7 +189,7 @@
 	vines += newVine
 	if(isliving(the_target))
 		var/mob/living/L = the_target
-		L.Paralyze(20)
+		L.Knockdown(10)
```

Comment:
This will likely be completely useless against borgs. The vine can be often hard to aim. especially when it considers every atom a possible target that can invoke the CD.

Lower the cooldown, add some borg behavior, and make it only be able to target objects and mobs